static/src/javascripts/projects/common/modules/experiments/tests/userzoom-survey-message-mobile-v3.js

```diff
@@ -0,0 +1,86 @@
+define([
+    'common/utils/$',
+    'bean',
+    'bonzo',
+    'fastdom',
+    'common/utils/config',
+    'lodash/utilities/noop',
+    'common/views/svgs',
+    'common/utils/template',
+    'common/modules/ui/message',
+    'text!common/views/survey-message.html',
+    'common/utils/detect',
+    'common/modules/user-prefs',
+    'common/utils/cookies'
+], function (
+    $,
+    bean,
+    bonzo,
+    fastdom,
+    config,
+    noop,
+    svgs,
+    template,
+    Message,
+    messageTemplate,
+    detect,
+    userPrefs,
+    cookies
+) {
+    return function () {
+        this.id = 'UserzoomSurveyMessageMobileV3';
+        this.start = '2016-02-02';
+        this.expiry = '2016-02-07';
+        this.author = 'Nathaniel Bennett';
+        this.description = 'Segment the userzoom data-team survey for mobile';
+        this.audience = 0.3;
+        this.audienceOffset = 0.0;
+        this.successMeasure = 'Gain qualitative feedback via a survey';
+        this.audienceCriteria = '30% of UK visitors to article page, on mobile, that haven\'t seen the message previously';
```

Comment:
Yeah, he just wanted it to be 30% compared to the 20% on desktop, so I'll check with him but I'm guessing that's good